docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,343 @@
-## Questions
+## Questions and Quizzes
 
-**Question component consists of a question body, a hint and an answer.**
+Question and quiz components provide an easy way to test readers on the relevant content topic in the page.
 
-<include src="codeAndOutput.md" boilerplate >
+#### Introduction
+
+Question components (`<question>`) can be one of the following types: **Mcq**, **Checkbox** or **Text**.
+
+In all cases, note that content directly inserted in between `<question>...</question>` will be inserted into the **question body**.
+
+You can also insert markdown into the **header** or **hint box**, by using the `header` and `hint` attributes respectively. Click the hint button below to see how the hint box turns out!
+
+<include src="codeAndOutputSeparate.md" boilerplate >
 <variable name="highlightStyle">html</variable>
+<variable name="heading">Header and hint syntax</variable>
 <variable name="code">
-<question>
-  The question body. <md>:grey_question:</md>
-  <div slot="hint">
-    Question hint. <md>:crystal_ball:</md>
-  </div>
-  <div slot="answer">
-    Question answer. <md>:pencil:</md>
-  </div>
+<!-- Insert markdown into the header and hint using the respective attributes -->
+<question ... header="Which of the following is correct? (**Header**)" hint="Think out of the box! :fas-box:">
+
+  <!-- Anything you place directly under a question not in a slot is inserted into the question body! -->
+  <small>Adapted from [Daily Mail](https://www.dailymail.co.uk/femail/article-4702868/Can-pass-intelligence-test.html)
+  </small>
+  </pic>
+
+  (**Question Body**)
+
+  <!-- Several hidden checkbox q-option components explained later -->
```

Comment:
Let's remove these to keep the examples _pure_. They can be even be misinterpreted as part of the question display template precisely because a reader doesn't expect them to be added by the author.
![image](https://user-images.githubusercontent.com/1673303/88457773-cf478b00-cebb-11ea-80fa-d09045a78f84.png)